Heimdall/Ethereum/EtherRPC+TokenInfo.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  EtherRPC+TokenInfo.swift
+//  Heimdall
+//
+//  Created by Luis Reisewitz on 06.11.17.
+//  Copyright © 2017 Gnosis. All rights reserved.
+//
+
+import BigInt
+import Bond
+import ReactiveKit
+
+extension EtherRPC {
+    func tokenInfo(for address: String) -> Signal<Token, Error> {
+        let nameSignal = call(HumanStandardToken.Name.self, ofContractAt: address)
+            .map { $0.value }
+        let symbolSignal = call(HumanStandardToken.Symbol.self, ofContractAt: address)
+            .map { $0.value }
+        // We can use forcedInt here, as the return value is only Solidity.UInt8
+        let decimalsSignal = call(HumanStandardToken.Decimals.self, ofContractAt: address)
+            .map { $0.value.forcedInt }
+
+        return combineLatest(nameSignal, symbolSignal, decimalsSignal)
```

Comment:
this is doing 3 request, right? So if 1 request fails the combineLatest will also fail. You could do them in a bulk request

Reply:
As far as I can see, my RPC provider does not allow for bulk requests at the moment. Looking to change that.